sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/api/Microsoft.Azure.WebJobs.Extensions.ServiceBus.netstandard2.0.cs

```diff
@@ -82,6 +82,8 @@ public partial class ServiceBusOptions : Microsoft.Azure.WebJobs.Hosting.IOption
         public int MaxConcurrentCalls { get { throw null; } set { } }
         public int MaxConcurrentSessions { get { throw null; } set { } }
         public int MaxMessageBatchSize { get { throw null; } set { } }
+        public System.TimeSpan MaxWaitTime { get { throw null; } set { } }
```

Comment:
I wonder if we should consider calling this MaxBatchWaitTime?

Reply:
Yeah I do like that more, I think in EH it was more consistent to call it just MaxWaitTime. Maybe it should be MaxMessageBatchTime to match the MinMessageBatchSize and MaxMessageBatchSize properties